models/face_image_quality_assessment_ediffiqa/ediffiqa.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of OpenCV Zoo project.
+# It is subject to the license terms in the LICENSE file found in the same directory.
+#
+# Copyright (C) 2021, Shenzhen Institute of Artificial Intelligence and Robotics for Society, all rights reserved.
+# Third party copyrights are property of their respective owners.
```

Comment:
Please drop these two lines as it is not valid.

Reply:
I have removed the lines as requested.